crates/forge/src/run_tests/resolve_config.rs

```diff
@@ -121,6 +129,15 @@ fn replace_id_with_params(
                 block: block_id,
             })
         }
+        RawForkConfig::Overridden(overridden) => {
+            let fork_target_from_runner_config =
+                get_fork_target_from_runner_config(fork_targets, &overridden.name)?;
+
+            Ok(InlineForkConfig {
+                url: fork_target_from_runner_config.url.parse()?,
+                block: overridden.block,
+            })
+        }
```

Comment:
A tiny aesthetic `nit` since you can unpack the struct. Totally depending on your preferences:
```suggestion
        RawForkConfig::Overridden(OverridenForkConfig { name, block }) => {
            let fork_target_from_runner_config =
                get_fork_target_from_runner_config(fork_targets, &name)?;
                
            let url = fork_target_from_runner_config.url.parse()?;

            Ok(InlineForkConfig { url, block })
        }
```

Reply:
It looks great